nimbus/Cargo.toml

```diff
@@ -30,6 +30,7 @@ sha2 = "0.9"
 hex = "0.4"
 once_cell = "1"
 uniffi = "0.4"
+semver = "0.11.0"
```

Comment:
Since we are only really looking at supporting 1-2 versions of the schema at a time, and only really need to look at Major versions, I can probably pare this back down to doing a little string slicing.  The convenience of the crate is good, but I'm not sure it's worth that big of a hit.